src/caret_analyze/infra/lttng/records_provider_lttng.py

```diff
@@ -681,7 +682,7 @@ def _compose_intra_proc_comm_records(
 
         Parameters
         ----------
-        comm_info : CommunicationStructInfo
+        comm_info : CommunicationStructValueInfo
```

Comment:
こちらは元の実装が誤っていました。
変更していただけるのであれば、正しい方に直していただけると幸いです。
```suggestion
        comm_info : CommunicationStructValue
```

Reply:
対応しました。